Makefile

```diff
@@ -1,6 +1,9 @@
 install:
 	obt install --verbose
 
+verify:
+	nbt verify --skip-layout-checks
+
 test:
 	obt build
-	obt verify
+	nbt verify --skip-layout-checks
```

Comment:
we normally don't call build from verify because CI will run both…  but otherwise :+1: